go/integTest/shared_commands_test.go

```diff
@@ -1687,6 +1688,62 @@ func (suite *GlideTestSuite) TestSPop_LastMember() {
 	})
 }
 
+func (suite *GlideTestSuite) TestSUnion() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key1 := "{key}-1-" + uuid.NewString()
+		key2 := "{key}-2-" + uuid.NewString()
+		key3 := "{key}-3-" + uuid.NewString()
+		nonSetKey := uuid.NewString()
+		memberList1 := []string{"a", "b", "c"}
+		memberList2 := []string{"b", "c", "d", "e"}
+		expected1 := map[api.Result[string]]struct{}{
+			api.CreateStringResult("a"): {},
+			api.CreateStringResult("b"): {},
+			api.CreateStringResult("c"): {},
+			api.CreateStringResult("d"): {},
+			api.CreateStringResult("e"): {},
+		}
+		expected2 := map[api.Result[string]]struct{}{
+			api.CreateStringResult("a"): {},
+			api.CreateStringResult("b"): {},
+			api.CreateStringResult("c"): {},
+		}
+
+		res1, err := client.SAdd(key1, memberList1)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(3), res1.Value())
+		assert.False(suite.T(), res1.IsNil())
+
+		res2, err := client.SAdd(key2, memberList2)
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), int64(4), res2.Value())
+		assert.False(suite.T(), res2.IsNil())
+
+		res3, err := client.SUnion([]string{key1, key2})
+		assert.Nil(suite.T(), err)
+		assert.True(suite.T(), reflect.DeepEqual(res3, expected1))
+
+		res4, err := client.SUnion([]string{key3})
+		assert.Nil(suite.T(), err)
+		assert.Equal(suite.T(), map[api.Result[string]]struct{}(map[api.Result[string]]struct{}{}), res4)
```

Comment:
Oh what is that? An empty response?

Reply:
Possible solution?

assert.Equal(suite.T(), map[api.Result[string]]struct{}{}, res4)

Reply:
Yi-Pin's solution works, the test is still passing after this change.